request_handler.gemspec

```diff
@@ -23,8 +23,8 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.add_dependency 'dry-validation', '~> 0.11', '>= 0.11.0'
-  spec.add_dependency 'dry-types', '~> 0.11', '>= 0.11.0'
+  spec.add_dependency 'dry-validation', '~> 1.0'
```

Comment:
So remove the dependency here, make the validation engine config required (this will be the first required config then btw), and have devs add the correct dry version?

Reply:
yeah, solves the issue that people indirectly rely on the dry-* stuff from this gem while using it elsewhere and allows use of other engines without unnecessary dry dependencies

Reply:
Removed the dependency